backends/edk2-compat/edk2-svc-read.c

```diff
@@ -374,6 +374,8 @@ static int printReadable(const char *c, size_t size, const char *key)
 		if (key && !strcmp(key, "dbx")) {
 			printf("\tHash: ");
 			printHex(cert, cert_size);
+		} else if (key && !strcmp(key, "sbat")) {
+			printf("\tFound sbat Info:\n%s\n", cert);
```

Comment:
`cert` is not null terminated. So valgrind reports buffer over-read. To fix, use secvarctl function `printRaw`.

Reply:
Fixed it